src/sidebar/components/VersionInfo.js

```diff
@@ -9,6 +10,35 @@ import { withServices } from '../service-context';
  * @prop {import('../services/toast-messenger').ToastMessengerService} toastMessenger
  */
 
+/**
+ * @typedef VersionEntryProps
+ * @prop {boolean} [breakAll=false] - Should this entry break long strings of
+ *   text, even if they are mid-word? (This prevents long strings from breaking
+ *   layout but may cause awkward breaks in shorter text)
+ * @prop {string} field
+ * @prop {string} value
+ */
+
+/**
+ * Render an entry for one piece of version data.
+ *
+ * @param {VersionEntryProps} props
+ */
+function VersionEntry({ breakAll = false, field, value }) {
+  return (
+    <>
+      <dt className="font-medium col-span-1 justify-self-end">{field}</dt>
+      <dd
+        className={classnames('col-span-3 text-grey-6 ', {
```

Comment:
In real life, we'd likely add tetx-related colors to the tailwind config instead of using `grey-6` directly.